extensions/wikia/RTE/RTE.class.php

```diff
@@ -290,7 +293,7 @@ public static function removeDefaultToolbar(&$toolbar) {
 	/**
 	 * Add fake form used by MW suggest in CK dialogs
 	 */
-	public static function onSkinAfterBottomScripts($skin, &$text) {
+	public static function onSkinAfterBottomScripts( $skin, &$text ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$skin". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)